### PR TITLE
MGDAPI-5815 - add golangci-lint prow check to cro

### PR DESCRIPTION
**Pull request by kramaranya**
[MGDAPI-5815](https://issues.redhat.com/browse/MGDAPI-5815)

### CODE REVIEW

File: ci-operator/config/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-master.yaml [hunk 1]
```diff
@@ -92,6 +92,10 @@ tests:
   commands: make gosec
   container:
     from: src
+- as: lint
+  commands: make GOLANGCI_LINT_CACHE=/tmp/golangci-cache make test/lint
```
Comment (Patryk-Stefanski) [inline ci-operator/config/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-master.yaml:96]: I don't think we need this make command at the start 

```suggestion
  commands:  GOLANGCI_LINT_CACHE=/tmp/golangci-cache make test/lint
```

**Review by Patryk-Stefanski — COMMENTED** [review]
Just one small suggestion looks good other than that.

File: ci-operator/config/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-master.yaml (hunk 1 above)
Comment (kramaranya) [reply ci-operator/config/integr8ly/cloud-resource-operator/integr8ly-cloud-resource-operator-master.yaml]: Thanks for spotting it! Updated

**Pull request merged**